### PR TITLE
Buildlog analysis trigger

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Depends-On: https://github.com/thoth-station/messaging/pull/326
Depends-On: https://github.com/thoth-station/user-api/pull/1171
Depends-On: https://github.com/thoth-station/common/pull/1073
Depends-On: https://github.com/thoth-station/common/pull/1076

## This introduces a breaking change

- [x] No

### CODE REVIEW

**Comment by fridex** [comment]
Needs https://github.com/thoth-station/messaging/pull/330

File: thoth/investigator/build_analsysis_trigger/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# BuildAnalysisTriggerMessage
+
+[BuildAnalysisTriggerMessage](https://github.com/thoth-station/messaging/blob/master/thoth/messaging/build_analysis_trigger.py)
+
+Thoth investigator schedules the following workflow to provide advice to user (human or bot):
```
Comment (pacospace) [inline thoth/investigator/build_analsysis_trigger/README.md:5]: description need to be updated

**Review by pacospace — COMMENTED** [review]
One minor comment otherwise very good!! :100: Thanks!

File: thoth/investigator/build_analsysis_trigger/README.md (hunk 1 above)
Comment (fridex) [reply thoth/investigator/build_analsysis_trigger/README.md]: :eyeglasses: adjusted :+1:

**Review by pacospace — APPROVED** [review]
:beer:  :motor_boat:

**Review by KPostOffice — CHANGES_REQUESTED** [review]
`run_module("thoth.investigator.build_analysis_trigger.investigate_build_analysis_trigger")`

This line needs to be added to `consumer.py` otherwise the function won't be added to the handler lookup table

**Comment by fridex** [comment]
Added. PTAL @KPostOffice

**Comment by pacospace** [comment]
@harshad16 /retest not working?

**Comment by harshad16** [comment]
checking on the systems

**Comment by harshad16** [comment]
This is a hostname issue, system is not able to get the result to Github.
working on it, will respond here , once i m able to fix this.

**Comment by harshad16** [comment]
It's back to business. thanks for all the patience

**Comment by fridex** [comment]
> It's back to business. thanks for all the patience

Thanks! :+1:

**Pull request merged**